x/ecocredit/basket/msg_create.go

```diff
@@ -101,6 +102,8 @@ func validateDateCriteria(d *DateCriteria) error {
 		if startDateWindow.Seconds < 24*3600 {
 			return errBadReq.Wrap("date_criteria.start_date_window must be at least 1 day")
 		}
+	} else if yearsInThePast != 0 {
+		// continue
 	} else {
 		return errBadReq.Wrapf("unsupported date_criteria value %v", d)
```

Comment:
do we not allow no date_criteria to be selected? i know it can be nil, but if all cases are empty, i don't think we should error

Reply:
This is currently the case in v3.0. I was wondering the same thing. I can update in this pull request.